modules/setting/cors.go

```diff
@@ -21,9 +21,10 @@ var CORSConfig = struct {
 	Headers          []string
 	XFrameOptions    string
 }{
-	Enabled:       false,
```

Comment:
Uninitualized bool defaults to false, right? Maybe keep it for explicity reasons.

Reply:
No reason, there is also a `AllowSubdomain`, no need to fill every empty fields.